### ISSUE TITLE
Improve first startup UI flow

### ISSUE THREAD

**Issue by Arthur-D** [supertuxkart/stk-code#1725]
To illustrate, I am going to provide screenshots of how things are currently and explain how I'd like them to be.
Upon starting the game for the first time, you are greeted with the internet and statistics information dialogue. When you click yes, you find this screen:

![create_user_screen](https://cloud.githubusercontent.com/assets/5515261/5116766/41584bea-704d-11e4-8911-ee313049505d.jpg)
There you are asked to either create an online user, or uncheck it to either not go online at all, or to log in with an existing user. Next screen:

![login_screen](https://cloud.githubusercontent.com/assets/5515261/5116820/ab2c8b44-704d-11e4-8c67-348ec36aa4e2.jpg)
Here, you have to check the Online checkbox to then proceed with logging in. This is in my opinion a bit convoluted and the opposite way around as to all games I've seen with optional online accounts. First screen they ask you to log in with an existing account, or optionally create a new one. 

To sum up: currently in STK, we ask them to create a new one, or uncheck something to log in the following screen. Users who don't read the instructions (and there are unfortunately a lot of them) may have an account but create a new one since they don't readily see the option to use it. Especially if they are not aware of the coupling of our addons site with the game, they may forget about the account they created for the addons. So I propose to have the login screen first, with a cancel button and a button "Create online account" which will take users to the account creation screen.

**Comment by math07** [comment]
I think the new menu in git latest version is a great improvement :)

**Comment by hiker** [comment]
This should be fixed now with the new register screen.
